trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -48,7 +49,12 @@ const getPartyUrn = async ({ companyRegNo, companyName }) => {
       return '';
     }
 
-    partyDbInfo = await api.getOrCreatePartyDbInfo({ companyRegNo, companyName });
+    if (!probabilityOfDefault) {
```

Comment:
If we are simply getting a customer who already exists then we shoudn't need to provide `probabilityOfDefault`?

Reply:
Do we not default PoD to `14.1` if none specified, wondering if this business requirement has been captured?